src/test/java/com/lmax/disruptor/alternatives/MultiProducerSequencerVarHandle.java

```diff
@@ -112,9 +112,9 @@ public long next()
      * @see Sequencer#next(int)
      */
     @Override
-    public long next(final int n)
+    public long next(final int numberOfEntities)
```

Comment:
I think this renaming is very suspicious. Why did you rename the parameter here and not in the interface and other implementing types? Also, why did you rename `n` in `next` and not in `tryNext`? Moreover, `numberOfEntities` is a totally generic name that is not related to the Disruptor: `numberOfSequences` or `numberOfEvents` would have been better. Anyway, I do not think renaming this parameter is a good idea.